spec/helpers/sessions_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
```

Comment:
Let's remove these helper spec files since we're not going to implement them

Reply:
its a useless safety net i agree

Reply:
good point, I'll remove it and update the PR